README.md

```diff
@@ -98,7 +98,7 @@ with this waiver of copyright interest.
 
 > This repo is built on top of the
 [Angular2 Webpack Starter](https://github.com/AngularClass/angular2-webpack-starter)
-from Angular Class and includes the original [MIT License](/LICENSE). Should you
+from Angular Class and includes the original [MIT License](https://opensource.org/licenses/MIT). Should you
```

Comment:
I think this should be changed to point to the [AngularClass License.txt](https://github.com/presidential-innovation-fellows/code-gov-web/blob/master/AngularClass%20License.txt) file in the repo. That way the original copyright information is included.